src/debugger/nodeDebugWrapper.ts

```diff
@@ -250,4 +252,21 @@ export class NodeDebugWrapper {
     private isNullOrUndefined(value: any): boolean {
         return typeof value === "undefined" || value === null;
     }
+
+    /**
+     * Parses settings.json file for workspace root property
+     */
+    private getProjectRoot(args: any): string {
+        try {
+            let vsCodeRoot = path.resolve(args.program, "../..");
+            let settingsPath = path.resolve(vsCodeRoot, ".vscode/settings.json");

```

Comment:
Well yes, I guess the entry point is still in `root/.vscode/`
If that is not the right place for it to be and we would place it to `root/project/.vscode`, then the user would need to manually edit `program` arg in `launch.json`.
Currently it looks like this:
```
{
<...>
    "configurations": [
        {
            ...
            "program": "${workspaceRoot}/.vscode/launchReactNative.js",
            ...
        }]
<...>
}
```
And as far as I know we can't really use any other variable other than `$workspaceRoot` here so we don't have any convenient way to point to project root here.